src/libxrpl/protocol/TER.cpp

```diff
@@ -115,6 +115,7 @@ transResults()
         MAKE_ERROR(tecTOKEN_PAIR_NOT_FOUND,          "Token pair is not found in Oracle object."),
         MAKE_ERROR(tecARRAY_EMPTY,                   "Array is empty."),
         MAKE_ERROR(tecARRAY_TOO_LARGE,               "Array is too large."),
+        MAKE_ERROR(tecBAD_CREDENTIALS,               "Bad credentials: not exists / not authorized / doesn't belong to account."),
```

Comment:
I don't like the use of slashes - can we make this a more general text, so that it's also future proof? Maybe just `Bad credentials`? cc @mDuo13 who I suspect has opinions on this.

Reply:
Fixed